compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Repl.hs

```diff
@@ -83,6 +84,8 @@ renderError dflags err = case err of
         putStrLn (showSDoc dflags err)
     (UnsupportedStatement str) ->
         putStrLn ("Unsupported statement: " <> str)
+    (UnknownModules names) ->
```

Comment:
Surprised hlint doesn’t complain about the redundant parentheses here. (I never cared about that rule so no need to change anything).